### ISSUE TITLE
SpectrumList_Thermo FAIMS compensation voltage parameter is placed in the wrong location semantically.

### ISSUE THREAD

**Issue by mobiusklein** [ProteoWizard/pwiz#747]
The SpectrumList_Thermo implementation places the `MS_FAIMS_compensation_voltage` cvParam on the `Spectrum` object [(source)](https://github.com/ProteoWizard/pwiz/blob/96421c9ae8bd201ceb8f26b069732c78fa4ab47a/pwiz/data/vendor_readers/Thermo/SpectrumList_Thermo.cpp#L368). 

According to the PSI-MS vocabulary ["FAIMS compensation voltage"](https://github.com/HUPO-PSI/psi-ms-CV/blob/master/psi-ms.obo#L10014) is an `ion-mobility attribute` -> `scan attribute`, which means it shouldn't be on the `Spectrum`.

SpectrumList_Agilent puts the drift time under the `Scan` [(source)](https://github.com/ProteoWizard/pwiz/blob/22ec844b8f2dc7a5e98e8db40792c0ec68fc1e71/pwiz/data/vendor_readers/Agilent/SpectrumList_Agilent.cpp#L239), as does Waters [(source)](https://github.com/ProteoWizard/pwiz/blob/22ec844b8f2dc7a5e98e8db40792c0ec68fc1e71/pwiz/data/vendor_readers/Waters/SpectrumList_Waters.cpp#L223), and Bruker [(source)](https://github.com/ProteoWizard/pwiz/blob/22ec844b8f2dc7a5e98e8db40792c0ec68fc1e71/pwiz/data/vendor_readers/Bruker/SpectrumList_Bruker.cpp#L263).

Should the FAIMS compensation voltage be moved to the `Scan`, if so, should it be fixed just in case people have already built on the assumption that the FAIMS drift time parameter is located on the `Spectrum`?

If it *should* be fixed, I can try to do this, though I've never built ProteoWizard from source successfully before.

**Comment by chambm** [comment]
You're right. It should be fixed. In other cases like this I've just made the fix and let things fall where they may since it's pretty trivial to fix in downstream code. So I'll probably do that here as well.
